packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -82,6 +82,20 @@ const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplicati
 });
 ```
 
+You can also specify `stackId` to override the default value:
+
+```ts
+const app = new App();
+const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplication', {
+    applicationName: 'MyAssociatedApplication',
+    stackId: 'MyAssociatedApplicationStack',
+    stackProps: {
+        stackName: 'MyAssociatedApplicationStack',
```

Comment:
Can we not use stackName for the stackId instead of adding another prop? This seems redundant.

Reply:
We definitely can, although it will cause stack replacement for existing customers. Given that this is alpha, would you be okay that I'll make that change?

Reply:
@TheRealAmazonKendra if we use `stackName` for stack ID, does it require a dedicated test?

Reply:
Breaking changes are fine, just make sure that the fact that it's breaking shows up in the title. From your perspective, would using the stackName satisfy your use case and also not require extra fields that the user has to care about?

Reply:
And I'd say that as long as the change shows up in integration tests and impacts the outputs in a way we see when you run the tests, I don't _think_ any additional ones should be needed. But, I'd ask that you change the name of the stack in the test below so we can see the impact.